### PR TITLE
Fix #1059: Replace "Inbox" by "Notifications" in navigation

### LINKED ISSUE #1059: Replace "Inbox" by "Notifications" in navigation
Change "Inbox" in "Notifications"
Change the icon to "bell"
Keep the "Coming soon" badge

# Current behavior
image

# Desired behavior
image

### PR DESCRIPTION
**Pull request by ducksblock**
### This fixes #1059

- Changed the label to "Notifications"
- Changed the icon to IconBell

### CODE REVIEW

File: front/src/AppNavbar.tsx [hunk 1]
```diff
@@ -16,6 +15,7 @@ import { useIsSubMenuNavbarDisplayed } from '@/ui/layout/hooks/useIsSubMenuNavba
 import MainNavbar from '@/ui/navbar/components/MainNavbar';
 import NavItem from '@/ui/navbar/components/NavItem';
 import NavTitle from '@/ui/navbar/components/NavTitle';
+import { IconBell } from '@tabler/icons-react';
```
Comment (jessicali2023) [inline front/src/AppNavbar.tsx:18]: "import" probably needs to be in alphabetic order.

Comment (FelixMalfait) [reply front/src/AppNavbar.tsx:18]: Yes good point @jessicali2023! @ducksblock you can run something like `npx eslint src/ --fix` in the `front` folder to do those fixes automatically

Comment (charlesBochet) [reply front/src/AppNavbar.tsx:18]: @ducksblock Also, please import icons from "@/ui/icon" (see other icons for example)

**Review by charlesBochet — COMMENTED** [review]
Thanks a lot @ducksblock!
Could you please take the comments into account? :)

**Review by charlesBochet — APPROVED** [review]
Neat, thank you!

**Pull request merged**